gridstatus/ercot.py

```diff
@@ -1773,6 +1777,50 @@ def _make_bid_curve(df):
 
         return self.parse_doc(df, verbose=verbose)
 
+    @support_date_range("DAY_START")
+    def get_sced_system_lambda(self, date, verbose=False):
+        # ANCHOR - SCED System Lambda
+        """Get System lambda of each successful SCED
+
+        5 Minute Publish Interval
+
+        https://github.com/kmax12/gridstatus/issues/269
+
+        Arguments:
+            date (str, datetime): date to get data for
+            verbose (bool, optional): print verbose output. Defaults to False.
+
+        Returns:
+            pandas.DataFrame: A DataFrame
+
+        """
+        if date == "latest":
+            doc = self._get_document(
+                report_type_id=SCED_SYSTEM_LAMBDA_RTID,
+                date=date,
+                verbose=verbose,
+            )
+            df = pd.read_csv(doc.url, compression="zip")
```

Comment:
maybe just do `docs = [doc]` here and then have shared logic for reading the csv

Reply:
I usually prefer more verbose logic, but changed as well